app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb

```diff
@@ -8,6 +8,7 @@ def validate_detach_volume
   end
 
   def raw_attach_volume(server_ems_ref, device = nil)
+    device = nil if (device && device.empty?)
```

Comment:
@sseago See the bot comment in this thread, which says the same thing.  But yeah, I prefer the `device.try` thing anyway...if you  don't get a response, just do the second, since we both like it better :)